docs/CONFIGURING-SLACK.md

```diff
@@ -26,63 +33,37 @@
  - users:read.email
 ```
 
-### User Token Scopes
-```
-- channels:read
-```
```

Comment:
This scope is still needed
![image](https://user-images.githubusercontent.com/18470453/91316085-d0a3f600-e78e-11ea-8b1b-89dd1bbeb9bd.png)

Reply:
In all the tests I made this configuration was not necessary. Would you know where this configuration is used?

Reply:
When I did the migration to the new scopes this scope was obligatory, but I'm also not sure where it's used. Production and staging have it and we can delete it. In your tests, you had `events` configured? I think both things are related.

Reply:
We are no longer using User Token, in all configurations we are using only the Bot Token